packages/turbo-repository/rust/src/lib.rs

```diff
@@ -63,4 +82,18 @@ impl Workspace {
     pub async fn find_packages(&self) -> std::result::Result<Vec<Package>, napi::Error> {
         self.packages_internal().await.map_err(|e| e.into())
     }
+
+    #[napi]
+    pub async fn package_graph(&self) -> HashMap<String, Vec<String>> {
```

Comment:
I would like to just return the whole Package here instead of the names, but (1) not sure if that's necessary and (2) not sure how to implement `ToNapiValue` trait so it can get into JS land.

Reply:
The `ToNapiValue` trait is only implemented for `HashMap`'s that have a key type that can be [referenced as a `str`](https://docs.rs/napi/latest/napi/bindgen_prelude/trait.ToNapiValue.html#impl-ToNapiValue-for-HashMap%3CK,+V,+S%3E). If you want the full package information available for each entry you might need to return something like
`HashMap<String, Node>` where `struct Node { package: Package, deps: Vec<Package> }`

Reply:
I'll just do relative paths for now, I _think_ that's the only thing we need right now. Will be taking multiple passes at this over the next couple days I'm sure